src/components/AdminPage/ConfirmDialog.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Confirm } from 'semantic-ui-react';
+
+class ConfirmDialog extends Component {
+  constructor(props) {
+    super(props);
+    this.toggleDialog = this.toggleDialog.bind(this);
+    this.deleteArticle = this.deleteArticle.bind(this);
+    this.state = { isOpen: false };
+  }
+
+  deleteArticle() {
+    const { id } = this.props;
+    this.deleteArticle(id);
```

Comment:
For the delete article function i created it so it can be called once you press confirm. Once you press confirm it should pass the id to the deleteArticle function